lib/facebook/messenger/welcome.rb

```diff
@@ -0,0 +1,61 @@
+require 'httparty'
+
+module Facebook
+  module Messenger
+    # This module handles setting welcome messages.
+    #
+    # https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message_configuration
+    module Welcome
+      include HTTParty
+
+      base_uri 'https://graph.facebook.com/v2.6/me'
```

Comment:
Yeah! me is a shorthand for the page that the given access token belongs to, though. That way, we never even have to ask which page it is.